lib/ramble/ramble/spack_runner.py

```diff
@@ -676,6 +676,36 @@ def mirror_environment(self, mirror_path):
   %-4d added
   %-4d failed to fetch.""" % (0, 0, 0)
 
+    def get_env_hash_list(self):
+        self._check_active()
+        args = [
+            'find',
+            '--format',
+            '/{hash}'
+        ]
+        output = self.spack(*args, output=str).strip()
+        return output
+
+    def push_to_spack_cache(self, spack_cache_path):
+        """Push packages for a given env to the spack cache"""
+        self._check_active()
+
+        hash_list = self.get_env_hash_list()
+
+        args = [
+            "buildcache",
+            "push",
+            "-a",  # TODO: remove this, it's just here for testing
```

Comment:
I think we'll just need a way to pass arguments in to push. Maybe the same way we pass in args for install and concretize?

Reply:
Good idea, done